From 06fea5c39b186955194e85ada50724465c39033d Mon Sep 17 00:00:00 2001
From: David Seifert <soap@gentoo.org>
Date: Sat, 10 Jun 2017 15:48:00 +0200
Subject: [PATCH] Remove htmldocdir variable

* Autoconf and Automake provide a variable (htmldir)
  for this already.

* asturm 2020-09-04: Rebased onto 0.1.9
---
 configure.ac    | 10 ----------
 doc/Makefile.am |  6 +-----
 2 files changed, 1 insertion(+), 15 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0ce20e9..6d5e692 100644
--- a/configure.ac
+++ b/configure.ac
@@ -264,21 +264,11 @@ AS_CASE([$target_os],
 		;;
 	esac
 
-htmldocdir=$prefix/share/doc/libsamplerate0-dev/html
-
-if test $prefix = "NONE" ; then
-	htmldocdir=/usr/local/share/doc/libsamplerate0-dev/html
-else
-	htmldocdir=$prefix/share/doc/libsamplerate0-dev/html
-	fi
-
 #====================================================================================
 # Now use the information from the checking stage.
 
 AC_DEFINE_UNQUOTED(OS_IS_WIN32, ${os_is_win32}, [Set to 1 if compiling for Win32])
 
-AC_SUBST(htmldocdir)
-
 AC_SUBST(SHLIB_VERSION_ARG)
 AC_SUBST(SHARED_VERSION_INFO)
 AC_SUBST(OS_SPECIFIC_INCLUDES)
diff --git a/doc/Makefile.am b/doc/Makefile.am
index be86d64..cbc9963 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -1,7 +1,3 @@
-htmldir = $(htmldocdir)
-html_DATA = SRC.png SRC.css index.html license.html history.html download.html \
+dist_html_DATA = SRC.png SRC.css index.html license.html history.html download.html \
 				lists.html quality.html win32.html faq.html api.html \
 				api_simple.html api_callback.html api_full.html api_misc.html
-
-EXTRA_DIST = $(html_DATA)
-
